doc/api/repl.markdown

```diff
@@ -226,10 +229,13 @@ There are a few special REPL commands:
     >.save ./file/to/save.js
   - `.load` - Load a file into the current REPL session.
     >.load ./file/to/load.js
+  - `.scope` - Show both global and local(scope) variables
```

Comment:
It doesn't actually work this way, it is sort of an internal command. I'd rather say that it could be passed to the `eval` function and should be accepted by the user handler.